sidhariyal/asterism_lines.fab

```diff
@@ -0,0 +1,96 @@
+# Ray helpers (no asterisms as fact)

```

Comment:
Are those asterisms (figures outside the IAU88 canonized constellations) genuine Tamil or 1990s Western tradition? In the latter case, just omit the asterism* files.

Reply:
Thanks & Understood, will follow accordingly.